src/setup/kmd/rest_api/client.rs

```diff
@@ -75,4 +79,50 @@ impl ClientV1 {
                 )
             })
     }
+
+    /// Get the list of public keys in the wallet.
+    pub async fn get_keys(&self, wallet_handle: String) -> anyhow::Result<ListKeysResponse> {
```

Comment:
I'd rename the arg to `wallet_handle_token`. Clearly, this is what it is.

Reply:
Done